.travis.yml

```diff
@@ -42,6 +43,10 @@ matrix:
   allow_failures:
     - php: hhvm
 
+install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
```

Comment:
@greg0ire Wouldn't it be better to have a something like a `.travis` directory and put shell scripts inside ?